src/library_gl.js

```diff
@@ -1400,6 +1411,97 @@ var LibraryGL = {
     GLctx.bufferSubData(target, offset, HEAPU8.subarray(data, data+size));
   },
 
+  // Queries EXT
```

Comment:
Alright, looks like duplicating all the functions is, sadly, the best way to go here, we do want to strongly segregate the APIs and objects here. In the future if the WebGL spec extensions end up explicitly noting that this doesn't need to be the case, then we can fuse them together. Great work for being so specific with this aspect, this would have fallen through my attention completely otherwise.